lib/spack/spack/test/cmd/pkg.py

```diff
@@ -187,7 +187,6 @@ def test_pkg_removed(mock_pkg_git_repo):
     assert out == ["pkg-c"]
 
 
-@pytest.mark.not_on_windows("stdout format conflict")
```

Comment:
@johnwparent I think if this test had run on windows, it may have caught the issue with untouched spec pruning on that platform.  I'm just curious if you know the history behind why some of the tests in here are marked `not_on_windows`?

Reply:
Since we have CI running, I think it makes sense to have the CI unit-test modules as enabled as possible. Lets get this fix in (without the test) so we have it functioning in CI and make the CI unit tests a priority for Spack on Windows (i.e. I'll get that enabled ASAP)